video/graphics.h

```diff
@@ -596,7 +609,9 @@ void plotCharacter(char c) {
 			canvas->drawChar(activeCursor->X, activeCursor->Y, c);
 		}
 	}
-	cursorRight();
+	if (!cursorBehaviour.xHold) {
+		cursorRight(cursorBehaviour.scrollProtect);
+	}
```

Comment:
Quark's documentation claimed that the cursor behaviour bit 5, the x-movement control which when set would prevent the cursor moving right after printing a character, was supported.

as can be seen tho, the `plotCharacter` would always call `cursorRight`, therefore it was not actually supported at all.  it is now.  :grin: